cirq-core/requirements.txt

```diff
@@ -13,6 +13,6 @@ numpy~=1.16
 pandas
 sortedcontainers~=2.0
 scipy
-sympy
+sympy==1.9
```

Comment:
Should we link the bug to fix type errors here?

Reply:
Done.